server/src/validator.js

```diff
@@ -57,7 +59,8 @@ export const byReqKey = {
 	},
 	term(value) {
 		const termWrapper = typeof value == 'string' ? JSON.parse(value) : value
-		if (!('id' in termWrapper)) throw 'missing termWrapper.id'
+		if (!('id' in termWrapper) && termWrapper.term?.type && isDictionaryType(termWrapper.term.type))
```

Comment:
just delete this check. it's based on legacy rule (requires tw.id but it's no more), enforces misleading name (term vs tw), too assumptive and doesn't cover other tw that could come from same request